### PR TITLE
Create Dotnet9 image

### PR DESCRIPTION
**Pull request by CooperLink**
Add Dotnet9 images. they work in CI. Release with 4.1035.1

### CODE REVIEW

File: host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    dotnet publish -v q /p:NugetAudit=false /p:BuildNumber=$BUILD_NUMBER /p:CommitHash=$HOST_COMMIT src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj -c Release --output /azure-functions-host --runtime linux-x64 --framework net8.0 --self-contained /p:MinorVersionPrefix=8 && \
+    mv /azure-functions-host/workers /workers && mkdir /azure-functions-host/workers && \
+    rm -rf /root/.local /root/.nuget /src
+
+FROM mcr.microsoft.com/dotnet/runtime-deps:9.0-preview-bookworm-slim-amd64
+ARG HOST_VERSION
+
+ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
+    HOME=/home \
+    FUNCTIONS_WORKER_RUNTIME=dotnet \
```
Comment (kshyju) [inline host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile]: `dotnet-isolated` ?

Comment (CooperLink) [reply host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile]: Oh yeah thanks. I will make sure that it works as intended when I have latest host/worker avail

File: host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Build the runtime from source
+ARG HOST_VERSION=4.834.4
```
Comment (pragnagopa) [inline host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile]: @CooperLink - update version to 4.1034.2

**Review by pragnagopa — CHANGES_REQUESTED** [review]
Update host version and include dotnet 9 image in all the publish files and publish the PR

File: host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile (hunk 2 above)
Comment (pragnagopa) [reply host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile]: @CooperLink - updated host version 4.1035.0

File: host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile (hunk 1 above)
Comment (CooperLink) [reply host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile]: Updated

File: host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Build the runtime from source
```
Comment (kshyju) [inline host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile:1]: Do we expect to add bookworm versions of other .net isolated versions (DNI10 or DNI8). If so, should we consider moving the files under "dotnet-isolated9" directory? I see we did that for the bullseye images.

File: host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+EXPOSE 2222 80
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends openssh-server dialog && \
+    echo "root:Docker!" | chpasswd && \
+    chmod +x /azure-functions-host/start.sh && \
+    chmod +x /opt/startup/install_ca_certificates.sh
+
+ENTRYPOINT ["/azure-functions-host/start.sh"]
```
Comment (kshyju) [inline host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile:41]: bullseye\amd64\dotnet\dotnet8-isolated has a `-slim` version of docker file as well. Do we need that here for net9?

Comment (CooperLink) [reply host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile:41]: No, -slim we should stop worrying about slim for this IMO.

Comment (CooperLink) [reply host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile:41]: We don't really provide them to customers. I added node 22 without it

File: host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile (hunk 3 above)
Comment (CooperLink) [reply host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile:1]: We could do that in the future. We are moving from this repo tho so lets not worry about it

**Review by pragnagopa — APPROVED** [review]

**Review by kshyju — APPROVED** [review]

**Pull request merged**